android/gradle.properties

```diff
@@ -1,6 +1,5 @@
 org.gradle.jvmargs=-Xmx1536M
 android.useAndroidX=true
 android.enableJetifier=true
-android.defaults.buildfeatures.buildconfig=true
```

Comment:
Why was this deleted?

Reply:
Deprecated as of Android Gradle Plugin 8.2.2